architecture/customizations.adoc

```diff
@@ -178,7 +178,7 @@ that are based on many of these CRDs to add more functionality to your
 |
 
 |Console
-|console.openshift.io
+|config.openshift.io
```

Comment:
So, this is confusing with all the operators.  I'll dump a little here:

```bash
# there are two configs, both named "cluster"
# all of the operators should follow this pattern:
oc describe console.config.openshift.io cluster
oc describe console.operator.openshift.io cluster

# this is the user-facing top level config that should be documented
console.config.openshift.io 

# this is the operator config, probably does not need to be called out 
console.operator.openshift.io
```
All `configs` for all `operators` are named `cluster` because they are intended to be singletons.

Reply:
`Can deploy CR`... console CR will be created by default with more or less empty values.  It will honor new values, and if its deleted it will recreate automatically.

Reply:
`Description` perhaps is `The top level configuration for the web console.`?  That may not be super helpful as you could say `The top level configuration for <component>` for all of them, so perhaps more detail is needed.